default_warehouse_from_sale_team/tests/test_default_warehouse.py

```diff
@@ -123,3 +126,57 @@ def test_04_warehouse_team_sale_policy(self):
         pick.action_assign()
         pick.move_ids.write({"quantity": 1, "picked": True})
         pick.button_validate()
+
+    def test_05_validate_saleteam_journal_access(self):
+        """Enforces record rule: move must use a public journal or one from the user's sales team;
+        otherwise AccessError with custom message.
+        """
+        today = fields.Date.context_today(self.env.user)
+        public_journal = self.env["account.journal"].create(
+            {
+                "name": "Public Sales",
+                "type": "sale",
+                "code": "PUB",
+            }
+        )
+        invoice = self.env["account.move"].create(
+            {
+                "move_type": "out_invoice",
+                "partner_id": self.env.ref("base.res_partner_2").id,
+                "company_id": self.company.id,
+                "date": today,
+                "invoice_date": today,
```

Comment:
Most of these fields are not required to be provided, as they already match their default values. You probably need only `move_type` and `journal`. Since you don't plan to validate it, not even `partner_id` should be required